test/scaladoc/run/t8557.scala

```diff
@@ -15,10 +17,22 @@ object Test extends ScaladocModelTest {
       class A
     """
 
+  def scalaURL = "http://www.scala-lang.org/api/current/"
+
   // a non-canonical path to scala-library.jar should still work
-  // this is a bit fragile (depends on the current directory being the root of the repo ;
-  // ant & partest seem to do that properly)
-  def scaladocSettings = "-doc-external-doc build/pack/bin/../lib/scala-library.jar#http://www.scala-lang.org/api/current/"
+  override def scaladocSettings = {
+    val samplePath = getClass.getClassLoader.getResource("scala/Function1.class").getPath.replace('\\', '/')
+    val scalaLibPath = if(samplePath.contains("!")) { // in scala-library.jar
+    val scalaLibUri = samplePath.split("!")(0)
```

Comment:
Another victim of IntelliJ's broken formatter (doesn't indent the line after a `{` when followed by a line comment)